keyboards/gboards/gergoplex/keymaps/default/rules.mk

```diff
@@ -0,0 +1,19 @@
+#----------------------------------------------------------------------------
+# make gboards/gergoplex:default:flash
+# Make sure you have dfu-programmer installed!
+#----------------------------------------------------------------------------
+# Firmware options
+MOUSEKEY_ENABLE        = no
```

Comment:
This line to `yes` might fix the mouse issues.

Reply:
Oh 🙈 
Thank you for spotting this!